modules/api/src/main/java/io/fluo/api/config/FluoConfiguration.java

```diff
@@ -66,6 +66,7 @@
   // Administration
   private static final String ADMIN_PREFIX = FLUO_PREFIX + ".admin";
   public static final String ADMIN_ACCUMULO_TABLE_PROP = ADMIN_PREFIX + ".accumulo.table";
+  public static final String ADMIN_HDFS_ROOT_PROP = ADMIN_PREFIX + ".hdfs.root";
```

Comment:
Initially, I was not going add the property to the class because it's only needed by the config file.  However, there might be future uses for it.  I can go either way.